### PR TITLE
include: nuttx: nxstyle fixes

### PR DESCRIPTION
**Pull request by jerpelea**
## Summary
nxstyle fixes for various headers

## Impact
none

## Testing
local nxstyle run

### CODE REVIEW

File: include/nuttx/can.h [hunk 1]
```diff
@@ -1,8 +1,8 @@
-/************************************************************************************
+/****************************************************************************
  * include/nuttx/can/can.h
  *
- *   Copyright (C) 2008, 2009, 2011-2012, 2015-2017, 2019 Gregory Nutt. All rights
- *     reserved.
+ *   Copyright (C) 2008, 2009, 2011-2012, 2015-2017, 2019 Gregory Nutt.
```
Comment (acassis) [inline include/nuttx/can.h:4]: Hi @jerpelea Greg already submitted this ICLA to convert source code from BSD to Apache. So, should be good to include this modifications when submitting these fixes.

**Review by acassis — CHANGES_REQUESTED** [review]

File: include/nuttx/can.h (hunk 1 above)
Comment (jerpelea) [reply include/nuttx/can.h:4]: I will handle the license in future commits

**Review by acassis — APPROVED** [review]

File: include/nuttx/timers/rtc.h [hunk 2]
```diff
@@ -117,7 +117,7 @@
 #endif
 
 #if defined(CONFIG_RTC_ALARM) && !defined(CONFIG_RTC_NALARMS)
-#  define CONFIG_RTC_NALARMS 1
```
Comment (xiaoxiang781216) [inline include/nuttx/timers/rtc.h:120]: why change this?

File: include/nuttx/timers/arch_timer.h [hunk 3]
```diff
@@ -46,7 +46,7 @@ void up_timer_set_lowerhalf(struct timer_lowerhalf_s *lower);
 
 #else
 
-#  define up_timer_set_lowerhalf(lower)
```
Comment (xiaoxiang781216) [inline include/nuttx/timers/arch_timer.h:49]: revert the change

File: include/nuttx/timers/arch_rtc.h [hunk 4]
```diff
@@ -46,7 +46,7 @@ void up_rtc_set_lowerhalf(FAR struct rtc_lowerhalf_s *lower);
 
 #else
 
-#  define up_rtc_set_lowerhalf(lower)
+#define up_rtc_set_lowerhalf(lower)
```
Comment (xiaoxiang781216) [inline include/nuttx/timers/arch_rtc.h:49]: Please review all similar change and revert it.

Comment (jerpelea) [reply include/nuttx/timers/arch_rtc.h:49]: ok

File: include/nuttx/net/ip.h [hunk 5]
```diff
@@ -461,7 +461,7 @@ extern "C"
 #ifdef CONFIG_NET_IPv6
 #  define net_ipv6addr_cmp(addr1, addr2) \
    (memcmp(addr1, addr2, sizeof(net_ipv6addr_t)) == 0)
-#  define net_ipv6addr_hdrcmp(addr1, addr2) \
+#define net_ipv6addr_hdrcmp(addr1, addr2) \
```
Comment (xiaoxiang781216) [inline include/nuttx/net/ip.h:464]: revert

File: include/nuttx/net/mii.h [hunk 6]
```diff
@@ -721,13 +721,13 @@
...
-#  define MII_EXT_CNTRL_TEST5             (5 << MII_EXT_CNTRL_TEST_SHIFT)
-#  define MII_EXT_CNTRL_TEST6             (6 << MII_EXT_CNTRL_TEST_SHIFT)
-#  define MII_EXT_CNTRL_TEST7             (7 << MII_EXT_CNTRL_TEST_SHIFT)
+#  define MII_EXT_CNTRL_TEST1               (1 << MII_EXT_CNTRL_TEST_SHIFT)
+#  define MII_EXT_CNTRL_TEST2               (2 << MII_EXT_CNTRL_TEST_SHIFT)
+#  define MII_EXT_CNTRL_TEST3               (3 << MII_EXT_CNTRL_TEST_SHIFT)
+#  define MII_EXT_CNTRL_TEST4               (4 << MII_EXT_CNTRL_TEST_SHIFT)
+#  define MII_EXT_CNTRL_TEST5               (5 << MII_EXT_CNTRL_TEST_SHIFT)
+#  define MII_EXT_CNTRL_TEST6               (6 << MII_EXT_CNTRL_TEST_SHIFT)
+#  define MII_EXT_CNTRL_TEST7               (7 << MII_EXT_CNTRL_TEST_SHIFT)
```
Comment (xiaoxiang781216) [inline include/nuttx/net/mii.h:730]: fix alignment

File: include/nuttx/net/netdev.h [hunk 7]
```diff
@@ -149,7 +150,7 @@
 #  define NETDEV_TXERRORS(dev)
 #  define NETDEV_TXTIMEOUTS(dev)
 
-#  define NETDEV_ERRORS(dev)
+#define NETDEV_ERRORS(dev)
```
Comment (xiaoxiang781216) [inline include/nuttx/net/netdev.h:153]: revert

File: include/nuttx/net/ip.h (hunk 5 above)
Comment (jerpelea) [reply include/nuttx/net/ip.h]: done

File: include/nuttx/net/mii.h (hunk 6 above)
Comment (jerpelea) [reply include/nuttx/net/mii.h]: done

File: include/nuttx/net/netdev.h (hunk 7 above)
Comment (jerpelea) [reply include/nuttx/net/netdev.h]: done

**Review by xiaoxiang781216 — APPROVED** [review]
